waggle-dance-integration-tests/src/test/java/com/hotels/bdp/waggledance/metrics/MonitoringConfigurationIntegrationTest.java

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (C) 2016-2018 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.bdp.waggledance.metrics;
+
+import static org.junit.Assert.fail;
+import static org.mockito.Matchers.contains;
+
+import java.util.Arrays;
+import java.util.Set;
+import java.util.TreeSet;
+
+import org.junit.Rule;
+import org.junit.Test;
+
+import com.codahale.metrics.MetricRegistry;
+
+import com.hotels.bdp.waggledance.conf.GraphiteConfiguration;
+import com.hotels.bdp.waggledance.junit.ServerSocketRule;
+
+public class MonitoringConfigurationIntegrationTest {
+
+  public @Rule ServerSocketRule graphite = new ServerSocketRule();
```

Comment:
ServerSocketRule is in this maven module so we had to put the test here as well to avoid a cyclical dependency